Makefile

```diff
@@ -0,0 +1,28 @@
+.ONESHELL:
+SHELL := /bin/bash
+.SHELLFLAGS = -ec
+
+ROOT_DIR := $(CURDIR)
+KUDO_TOOLS_DIR := $(ROOT_DIR)/shared
+SPARK_OPERATOR_DIR := $(ROOT_DIR)/spark-on-k8s-operator
+
+KONVOY_VERSION ?= v1.1.5
+export KONVOY_VERSION
+
+CLUSTER_TYPE ?= konvoy
+
+cluster-create:
+	$(KUDO_TOOLS_DIR)/cluster.sh $(CLUSTER_TYPE) up
+	echo > $(CLUSTER_TYPE)-created
+
+cluster-destroy:
```

Comment:
just wondering: should we remove  `*-created` flags after a cluster is destroyed?

Reply:
thanks for the nit, added the cleanup